### DISCUSSION TITLE
PCP Combine in METPlus Error

### DISCUSSION THREAD

**Discussion by craigr3** [dtcenter/METplus#1827] [Configuration] [answered]
I have been having difficulty getting the PCPCombine SUM METPlus capability to work.  I am trying to add up 8 buckets of model 3h precip accumulation to get a 24hour amount.  In my PCPCombin_sum.conf, the resulting metplus_final.conf is attached.
![config_file](https://user-images.githubusercontent.com/35153906/191783917-9c3d14cb-ee10-419f-9274-148050f04d7d.png)

The error message I get is  attached.
![PCPCombine_log](https://user-images.githubusercontent.com/35153906/191784483-a58e8ebe-add2-4947-8fad-0e12d6d395a3.png)

Never Mind.  I switched to ADD and got it to work.

**Comment by JohnHalleyGotway** [comment, marked as answer]
@craigr3 glad you were able to figure this out. Yes, both the ADD and SUM options are available via METplus. In general, I'd recommend using the ADD option since I find it to be more intuitive and actually runs faster. The SUM command is relatively slow since it inspects the contents of each potential input file to find the data it needs. All that file I/O is slow.

I'll go ahead and mark this one as being answered.
